test/ProjectService.Data.UnitTests/CreateRoleRepositoryTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using LT.DigitalOffice.ProjectService.Data.Interfaces;
+using LT.DigitalOffice.ProjectService.Data.Provider;
+using LT.DigitalOffice.ProjectService.Data.Provider.MsSql.Ef;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+
+namespace LT.DigitalOffice.ProjectService.Data.UnitTests
+{
+    class CreateRoleRepositoryTests
+    {
+        private IDataProvider provider;
+        private IRoleRepository repository;
+
+        private DbRole role;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var dbOptionsProjectService = new DbContextOptionsBuilder<ProjectServiceDbContext>()
+                .UseInMemoryDatabase("RoleTest")
+                .Options;
+
+            provider = new ProjectServiceDbContext(dbOptionsProjectService);
+
+            repository = new RoleRepository(provider);
```

Comment:
I'm not sure if it's good to create a separate database and repository for each test. 
(DB) You can create it once, and then clear and fill it.
(Repository) Create it once.

Reply:
Why should not we create database for each tests? We must isolate each test case from each other as much as possible. And besides, we always called the method after the test TearDown()

Reply:
This can (20-23 lines) already be move to OneTimeSetUp

Reply:
In real requests, a database is not created for each request, so I don't see the point in this